common/execution_context.go

```diff
@@ -74,6 +75,8 @@ type ExecutionContext struct {
 	injectedScript api.JSHandle
 	vu             k6modules.VU
 
+	isMutex sync.RWMutex
```

Comment:
Let's move this above `injectedScript`, which is usually the convention to indicate what the mutex is protecting.

Reply:
Thanks! Resolved and rebased into the initial commmit.